core/src/main/scala/spinal/core/Area.scala

```diff
@@ -24,39 +24,30 @@ import spinal.core.Nameable.DATAMODEL_WEAK
 import spinal.core.internals.Misc
 import spinal.idslplugin.PostInitCallback
 
-
-
-
-
-/**
-  * Sometime, creating a Component to define some logic is overkill.
-  * For this kind of cases you can use Area to define a group of signals/logic.
+/** Sometime, creating a Component to define some logic is overkill. For this kind of cases you can use Area to define a group of signals/logic.
   *
-  * @example {{{
-  *     val tickConter = new Area{
-  *       val tick = Reg(UInt(8 bits) init(0)
-  *       tick := tick + 1
-  *     }
-  * }}}
-  *  @see  [[http://spinalhdl.github.io/SpinalDoc/spinal/core/area/ Area Documentation]]
+  * @example
+  *   {{{val tickConter = new Area{ val tick = Reg(UInt(8 bits) init(0) tick := tick + 1}}} }
+  * @see
+  *   [[http://spinalhdl.github.io/SpinalDoc/spinal/core/area/ Area Documentation]]
   */
 
-class Composite[T <: Nameable](val self : T, postfix : String = null) extends Area{
+class Composite[T <: Nameable](val self: T, postfix: String = null) extends Area {
   override def childNamePriority = Nameable.USER_WEAK
-  if(postfix == null) {
+  if (postfix == null) {
     setCompositeName(self, true)
   } else {
     setCompositeName(self, postfix, true)
   }
 }
 
-trait Area extends NameableByComponent with ContextUser with OwnableRef with ScalaLocated with ValCallbackRec with OverridedEqualsHashCode  {
-  def childNamePriority = DATAMODEL_WEAK
-  val _context = ScopeProperty.capture() //TODO not as heavy
-  def rework[T](body : => T): T = {
-    val oldContext = ScopeProperty.capture() //TODO not as heavy
+trait Area extends NameableByComponent with ContextUser with OwnableRef with ScalaLocated with ValCallbackRec with OverridedEqualsHashCode {
+  def childNamePriority                     = DATAMODEL_WEAK
+  val _context                              = ScopeProperty.capture() // TODO not as heavy
+  def rework[T](body: => T): T              = {
```

Comment:
So that's how multiline entries with alignment are handled in Scala. Interesting. Less ugly than in Java, I guess…